ruby/style.yml

```diff
@@ -42,16 +42,12 @@ Style/Copyright:
   Enabled: false
 Style/Documentation:
   Enabled: false
-Style/Encoding:
```

Comment:
This is obsolete:

``
Error: obsolete parameter EnforcedStyle (for Style/Encoding) found in /Users/tbrooks/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/bundler/gems/house_style-c861bad2e1d2/ruby/style.yml
Style/Encoding no longer supports styles. The "never" behavior is always assumed.
```